pkg/network/tracer/tracer.go

```diff
@@ -146,7 +150,7 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 			string(probes.TcpStatsMap):        {Type: ebpf.Hash, MaxEntries: uint32(config.MaxTrackedConnections), EditorFlag: manager.EditMaxEntries},
 			string(probes.PortBindingsMap):    {Type: ebpf.Hash, MaxEntries: uint32(config.MaxTrackedConnections), EditorFlag: manager.EditMaxEntries},
 			string(probes.UdpPortBindingsMap): {Type: ebpf.Hash, MaxEntries: uint32(config.MaxTrackedConnections), EditorFlag: manager.EditMaxEntries},
-			string(probes.HttpInFlightMap):    {Type: ebpf.Hash, MaxEntries: uint32(config.MaxTrackedConnections), EditorFlag: manager.EditMaxEntries},
+			string(probes.HttpInFlightMap):    {Type: ebpf.Hash, MaxEntries: uint32(maxHTTPInFlightEntries), EditorFlag: manager.EditMaxEntries},
```

Comment:
Should we just not add this entry if http monitoring is not enabled?

Reply:
The BPF loader will load every map in the object file, so our best shot at the moment is to edit the max entries option.

Reply:
But isn't the max_entries set to 0 for this map in the bpf code?

Reply:
It is but that value is considered invalid and results in an error. I'd actually vote for changing that `1` for all maps (in a follow up PR). @brycekahle any thoughts on that/reasons for using `0` as the default value? Was the motivation to sort of fail loudly?

Reply:
ah ok, you can't have 0 for max_entries

Reply:
> Was the motivation to sort of fail loudly if these constants are not edited

Yes, exactly.